MibTeX/src/de/mibtex/BibtexEntry.java

```diff
@@ -103,11 +105,11 @@ else if (authorList.size() > 2)
 			pdf += " " + year;
 		pdf += " " + title;
 		pdf += ".pdf";
-		return new File(BibtexViewer.PDF_DIR + toURL(pdf));
+		return new File(BibtexViewer.PDF_DIR, toURL(pdf));
 	}
 
 	public String getRelativePDFPath() {
-		return BibtexViewer.PDF_DIR_REL + getPDFPath().getName();
+		return new File(BibtexViewer.PDF_DIR_REL, getPDFPath().getName()).toString();
```

Comment:
Are you sure this is a refactoring?

Reply:
Yes. I added this change to be sure a slash "/" is added correctly when concatenating paths.
The constructor is combining the two arguments and assumes the first one is a directory:
https://docs.oracle.com/javase/7/docs/api/java/io/File.html#File(java.lang.String,%20java.lang.String)
The `toString()` method also takes care of using the current operating system's slashes ('\' vs '/').